### PR TITLE
LPS-164576 Don't return if the content is empty and reuse the other logic as fallback

### PR DESCRIPTION
**Pull request by ealonso**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
Hey @balazssk,
I need to update your logic a little bit since your pr have conflicts and the logic where moved to an another method.

File: modules/apps/layout/layout-impl/src/main/java/com/liferay/layout/internal/content/LayoutContentProviderImpl.java [hunk 1]
```diff
@@ -176,7 +181,11 @@ public String getLayoutContent(Layout layout, Locale locale) {
 				}
 			}
 
-			return _html.stripHtml(_getWrapper(content));
+			content = _getWrapper(content);
+
+			if (Validator.isNotNull(content)) {
```
Comment (balazssk) [inline modules/apps/layout/layout-impl/src/main/java/com/liferay/layout/internal/content/LayoutContentProviderImpl.java:186]: Hi @ealonso ,
If content is null and httpServletRequest is null as well, then line 191 will throw an NPE,
I think we don't need this if condition here

Comment (ealonso) [reply modules/apps/layout/layout-impl/src/main/java/com/liferay/layout/internal/content/LayoutContentProviderImpl.java:186]: You are right, thanks for letting me know

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**